### ISSUE TITLE
Retrieve Address on load

### ISSUE THREAD

**Issue by alexander-code** [sgruhier/typeahead-addresspicker#24]
Hi, 

congrats for one more time for your plugin. Great work.
I want to ask you if it's possible the following case. What I did so far is that by clicking on a link I get the current user lat, long. I load a new php where I Get the values and set map's lat long so the marker is properly appears on the map on the correct point. So, beside this, I'm wondering if I can trigger any function and fill in the input type field with the address based on the lat long values while the page is loaded? (like when you drag the marker and the field fills in with the address of that point)

Thanks in advance.

**Comment by sgruhier** [comment]
You could ask again google to get places at lat/lng but I'm wondering why you don't save in your DB more than lat/lng. You should store all info you have lat/lng/address/... Like this you don't have to request something

**Comment by alexander-code** [comment]
Sorry maybe I didn't explained it correctly. I described a case when a user wants to find where is he and save this point as his place. To do that I need two things. The lat long and the address. Then
the user could give a name to this place and store it into my DB. 

"Visualise" the steps:

1) User clicks on a link and I get the current Lat,Long <-- Done
2) Based on the Lat, Long we've just found, we want the address of this point while the page is loading (just like when you drag the marker) <-- Help :)
3) Finally, user types a name and save it to the DB <-- Done

I want to know how to accomplish the 2nd step.
Thank you.
